src/app/content/hooks/receivePageNotFoundId.ts

```diff
@@ -16,6 +19,9 @@ export const receivePageNotFoundIdHookBody: ActionHookBody<typeof receivePageNot
       return;
     }
   }
+  if (bookConfig?.retired) {
+    return Promise.reject(new BookNotFoundError(`Could not resolve uuid: ${bookId}`));
```

Comment:
why do you need to do this here?

Reply:
because we only know to 404 a page from an old book once we've checked it for redirects.